plc4j/drivers/iec-60870/src/test/java/org/apache/plc4x/java/iec608705104/ManualTest.java

```diff
@@ -37,10 +37,10 @@ public static void main(String[] args) throws Exception {
                 throw new RuntimeException("Subscription not supported");
             }
 
-            plcConnection.subscriptionRequestBuilder().addChangeOfStateTagAddress("all", "*").addPreRegisteredConsumer("all", plcSubscriptionEvent -> {
+            plcConnection.subscriptionRequestBuilder().addChangeOfStateTagAddress("all", "*").setConsumer(plcSubscriptionEvent -> {
```

Comment:
shouldn't the consumer be set then as 3rd argument in addEventTagAddress following that logic? otherwise that would semantical not the same